guides/documentation.md

```diff
@@ -0,0 +1,32 @@
+
+In line with the CFPB [Source Code Policy](https://github.com/cfpb/source-code-policy/), documentation should be made public and open source where possible. Exceptions to this policy include documentation containing sensitive or internal data or processes. CFPB contributors should consult their management for guidance on where to locate private documentation.
+
+Per the [Nearby](https://www.writethedocs.org/guide/writing/docs-principles/#nearby) principle of documentation, docs should be stored as close as possible to the code that they document.
+
+General standards and guides about software development should live in
+this repository
+([https://github.com/cfpb/development](https://github.com/cfpb/development)). This documentation is written in Markdown and browseable [on GitHub](https://github.com/cfpb/development#cfpb-development-guidelines).
+
+Documentation specific to
+[https://www.consumerfinance.gov](https://www.consumerfinance.gov) or
+the software that powers it should live in the [cfgov-refresh](https://github.com/cfpb/cfgov-refresh) repository. Documentation is written as [Markdown files](https://github.com/cfpb/cfgov-refresh/tree/master/docs) in that repository, converted to HTML by [MkDocs](https://www.mkdocs.org/), and served by GitHub Pages at [https://cfpb.github.io/cfgov-refresh/](https://cfpb.github.io/cfgov-refresh/).
+
+Documentation specific to other projects should live in that project's repository. Exceptions to this include repositories that only work within the context of a larger project, for example the cf.gov satellite apps. In that case it may be appropriate to locate documentation within the cfgov-refresh repository, as long as the satellite app contains a link pointing to the appropriate location.
+
+## Documentation definition of done
+
+Any new or modified documentation should meet the following criteria before being published:
+
+1. _Thoroughly reviewed:_ Ideally, documentation should be peer-reviewed by two people: one who is familiar with the material being covered, and one who is new to the material. This will ensure that documentation is appropriate for developers with different experience levels.
+1. _Tested:_ Any documentation that describes a process (for example, setting up a project) should be tested fully by a developer to verify its correctness. Consider defining and using user stories to guide this process (for example, "as a new member of the team, I need to get cf.gov running locally on my machine").
+1. _Properly located:_ Documentation should be located in an appropriate place. [See "Where to put documentation" above](#where-to-put-documentation) for additional detail.
+1. _Properly discoverable:_ Documentation should be linked to from an appropriate landing page (for example, this repository's [`README.md`](https://github.com/cfpb/development/blob/master/README.md) and, if possible, be discoverable through search ([for example](https://cfpb.github.io/cfgov-refresh/search.html?q=testing)).
```

Comment:
```suggestion
1. _Properly discoverable:_ Documentation should be linked to from an appropriate landing page (for example, [this repository's README](https://github.com/cfpb/development/blob/master/README.md) and, if possible, be discoverable through search (like [the cfgov-refresh docs search](https://cfpb.github.io/cfgov-refresh/search.html?q=testing), for example).
```

Reply:
Applied (with semantic line breaks -- thanks for teaching me [about that](http://sembr.org/)) in 6080ed4.